streampipes-client-python/streampipes_client/__init__.py

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
Do we want to name the package `streampipes_client` or rather `streampipesclient`?

Reply:
`streampipes-client`